### ISSUE TITLE
Special instructions: all PIXEL instructions

### ISSUE THREAD

**Issue by gladky** [cmsdaq/DAQExpert#195]
Special instruction questions for PIXEL found in shifter bulletin board.
- [ ] #177  
- [ ] #189
- [ ] #190 
- [ ] #191 
- [ ] #192 
- [ ] #193 
- [ ] #194

**Comment by bvormwald** [comment]
#177 this can be removed
#189 the DOC should be called when pixel is in the state RunningDegraded at any time (also at night, don't wait 12min)
#190 to be removed
#191 to be removed
#192 suggested changes are fine
#193 instructions to stay in the problematic state (if not in stable beams) and call the expert should be kept
#194 we don't expect any problem there, but the additional check does also not harm
#196 still up-to-date

Thanks for the collection of instruction and the update!

**Comment by hsakulin** [comment]
> #189 the DOC should be called when pixel is in the state RunningDegraded at any time (also at night, don't wait 12min)

Hello Benedikt,
do the ERROR or ACTION messages from the PIXEL FM (which are shown to the shifter in the Level-0 FM) during Running Degraded clearly state that the DOC should be called immediately?
Cheers,
  Hannes.

**Comment by bvormwald** [comment]
No, we are not using the messaging between PIXEL FM and L0-FM at the moment.

**Comment by hsakulin** [comment]
A subsystem may use the RunningDegraded state to attract the attention of the shift crew. The subsystem may change into this state from Running state, asynchronously. It may also change back to Running state, asynchronously. This state does not mean that the run has to be stopped. Depending on the problem detected it may be fine to continue running or some action may need to be taken. While in RunningDegraded state, a subsystem should provide an analysis of the problem, ideally with clear instructions to the shift crew in its ACTION_MSG or ERROR_MSG exported parameters. Additionally, subsystems may provide a reason (text) to the input changing their state to RunningDegraded (using input.setReason(String reason)). This reason will be displayed by the Level-0 FM in the Run History pane and in the Timeline.

**Comment by hsakulin** [comment]
Suggestion for Running Degraded (for all Subsystems - it would also cover the Pixel case).

Subsytem XXX is in RunningDegraded. Please check the message in the subsystem information section in the Level-0 FM for subsystem XXX. It should contain a description of the problem.  Use your common sense to decide whether a human expert needs to look at the problem or whether CMS can continue taking data despite the problem. If there is no problem description or if the description is unclear, call the DOC of subsystem XXX.
